lib/hexpm/repository/packages.ex

```diff
@@ -135,4 +135,21 @@ defmodule Hexpm.Repository.Packages do
     |> Repo.all()
     |> Enum.into(%{})
   end
+
+  def accessible_user_owned_packages(user, for_user) do
```

Comment:
I'm not really sure about the function's name though. Feel free to suggest better name :)

Reply:
Maybe `visible_packages_for_user`? Not sure.